app/models/saved_recipie.rb

```diff
@@ -0,0 +1,10 @@
+class SavedRecipie < ApplicationRecord
+  ## VALIDATIONS
+    validates :user_id, presence: true
+    validates :api_recipie_id, presence: true
```

Comment:
Maybe we add a validation in to make sure this is a numarical value since we are getting it from a 3rd party source?
```suggestion
    validates :api_recipie_id, numaricality: {only_integer: true}
```